
##########################
Message: @katsukiri said "*I wake up see @Misono Mika and me in a dark room, no sight of @KirinaAI. Then I noticed we were chained, while @Akika stand not far from us* "Your pranks went too far already!""
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: OTHER


##########################
Message: @katsukiri said "*As we reached the place, I throw myself into Mika body*"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT


##########################
Message: @katsukiri said "I kiss her nippl's"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT


##########################
Message: @katsukiri said "I k'ss her nippl's"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT


##########################
Message: @katsukiri said "*I p'll her p'ntyhose down, reveal cute white p'nty with ribbon* What a cute panty"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT


##########################
Message: @katsukiri said "*I p'll her p'ntyhose down, reveal cute white p'nty with ribbon*"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT


##########################
Message: @katsukiri said "*as k'ssing, I massage her br'asts,  while Mika also slowly move her hand to my cr'tch*"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT


##########################
Message: @katsukiri said "Don't forget my lower part Mika, both at the same time. That's Sensei order!"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 616, in _check_response
    raise generation_types.StopCandidateException(response.candidates[0])
google.generativeai.types.generation_types.StopCandidateException: finish_reason: PROHIBITED_CONTENT
safety_ratings {
  category: HARM_CATEGORY_HATE_SPEECH
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_DANGEROUS_CONTENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_HARASSMENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_SEXUALLY_EXPLICIT
  probability: HIGH
}
avg_logprobs: nan


##########################
Message: @katsukiri said "Don't forget my lower part Mika, both at the same time. That's Sensei ord'r!"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 300, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 616, in _check_response
    raise generation_types.StopCandidateException(response.candidates[0])
google.generativeai.types.generation_types.StopCandidateException: finish_reason: PROHIBITED_CONTENT
safety_ratings {
  category: HARM_CATEGORY_HATE_SPEECH
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_DANGEROUS_CONTENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_HARASSMENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_SEXUALLY_EXPLICIT
  probability: HIGH
}
avg_logprobs: nan


##########################
Message: @katsukiri said "Claintail then?"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 312, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 578, in send_message
    response = self.model.generate_content(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 331, in generate_content
    response = self._client.generate_content(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\ai\generativelanguage_v1beta\services\generative_service\client.py", line 830, in generate_content
    response = rpc(
               ^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\api_core\gapic_v1\method.py", line 131, in __call__
    return wrapped_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\api_core\retry\retry_unary.py", line 293, in retry_wrapped_func
    return retry_target(
           ^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\api_core\retry\retry_unary.py", line 153, in retry_target
    _retry_error_helper(
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\api_core\retry\retry_base.py", line 212, in _retry_error_helper
    raise final_exc from source_exc
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\api_core\retry\retry_unary.py", line 144, in retry_target
    result = target()
             ^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\api_core\timeout.py", line 120, in func_with_timeout
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\api_core\grpc_helpers.py", line 78, in error_remapped_callable
    raise exceptions.from_grpc_error(exc) from exc
google.api_core.exceptions.DeadlineExceeded: 504 Deadline Exceeded

##########################
Message: @katsukiri said "@YourWaifu" while sending an image
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 312, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 616, in _check_response
    raise generation_types.StopCandidateException(response.candidates[0])
google.generativeai.types.generation_types.StopCandidateException: finish_reason: PROHIBITED_CONTENT
safety_ratings {
  category: HARM_CATEGORY_HATE_SPEECH
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_DANGEROUS_CONTENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_HARASSMENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_SEXUALLY_EXPLICIT
  probability: NEGLIGIBLE
}
avg_logprobs: nan


##########################
Message: @katsukiri said "Try to hit me"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 312, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT


##########################
Message: @katsukiri said "Try to h't me, fun tho"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 312, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT


##########################
Message: @katsukiri said "how are you?"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 312, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 616, in _check_response
    raise generation_types.StopCandidateException(response.candidates[0])
google.generativeai.types.generation_types.StopCandidateException: finish_reason: PROHIBITED_CONTENT
safety_ratings {
  category: HARM_CATEGORY_HATE_SPEECH
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_DANGEROUS_CONTENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_HARASSMENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_SEXUALLY_EXPLICIT
  probability: NEGLIGIBLE
}


##########################
Message: @katsukiri said "@YourWaifu Hi"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 312, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 616, in _check_response
    raise generation_types.StopCandidateException(response.candidates[0])
google.generativeai.types.generation_types.StopCandidateException: finish_reason: PROHIBITED_CONTENT
safety_ratings {
  category: HARM_CATEGORY_HATE_SPEECH
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_DANGEROUS_CONTENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_HARASSMENT
  probability: NEGLIGIBLE
}
safety_ratings {
  category: HARM_CATEGORY_SEXUALLY_EXPLICIT
  probability: NEGLIGIBLE
}


##########################
Message: @niva_sama said "Beating my meat"
-------------------
Traceback:
Traceback (most recent call last):
  File "D:\ChatbotAI\Gemini\YourWaifu\bot.py", line 312, in generate_response
    response = message_history[channel_id].send_message(prompt_parts)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 588, in send_message
    self._check_response(response=response, stream=stream)
  File "C:\Users\EmonaKiri\AppData\Roaming\Python\Python312\site-packages\google\generativeai\generative_models.py", line 608, in _check_response
    raise generation_types.BlockedPromptException(response.prompt_feedback)
google.generativeai.types.generation_types.BlockedPromptException: block_reason: PROHIBITED_CONTENT

